ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml

```diff
@@ -2155,6 +2155,80 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-metal-ipi-ovn-ipv6,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
Should we change this to false and then we can trigger it manually when we think appropriate? Just conscious that there are a lot of presubmits on this repo already. Are there specific changes we could use to trigger it? Eg anything nutanix related we can make a `run_only_if_change` regex up for?

Reply:
Yep, we should make that change.  Thanks Joel.  I'll change it to `always_run: false` as I don't think we'll need to run this very often.